lib/Cargo.toml

```diff
@@ -64,7 +64,7 @@ url = "2.2.2"
 uuid = { version = "1.1.2", features = ["serde", "v4"] }
 
 [dev-dependencies]
-tokio = { version = "1.20.1", features = ["macros"] }
+tokio = { version = "1.20.1", features = ["macros", "rt"] }
```

Comment:
Hey @finnbear , just a question for my understanding...

This is obviously for the `#[tokio::test]` test below. However we already have async tokio tests within the code, so howcome it's necessary to add this? Not saying it isn't just confused...!

Reply:
I believe having the feature flag is more correct, as it is needed for running `cargo --test` the `/lib` crate. The only reason `cargo --test` worked in the root directory of the repository (what the CI cares about) is because (presumably) some other member of the repository (or one of its dependencies) adds the feature.

Considering this, let me know if you want to keep the feature flag (or sacrifice ability to `cargo --test` in `/lib`).

Reply:
Yeah that makes sense. No let's keep it so that tests in the lib work correctly on their own to!